src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -3,6 +3,15 @@
 Microsoft.CodeAnalysis.CommandLineArguments.EmitPdbFile.get -> bool
 Microsoft.CodeAnalysis.CommandLineArguments.GetOutputFilePath(string outputFileName) -> string
 Microsoft.CodeAnalysis.CommandLineArguments.GetPdbFilePath(string outputFileName) -> string
+*REMOVED*Microsoft.CodeAnalysis.Compilation.CreateTupleTypeSymbol(Microsoft.CodeAnalysis.INamedTypeSymbol underlyingType, System.Collections.Immutable.ImmutableArray<string> elementNames = default(System.Collections.Immutable.ImmutableArray<string>), System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Location> elementLocations = default(System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Location>)) -> Microsoft.CodeAnalysis.INamedTypeSymbol
```

Comment:
removing?

Reply:
The existing overloads have been replaced (rather than removed) by overloads with the same signature but no default parameter values. New overloads have been added that have default parameter values. Is that sufficient for compatibility?

---
In reply to: [301325829](https://github.com/dotnet/roslyn/pull/37048#discussion_r301325829) [](ancestors = 301325829)

Reply:
Ah gotcha.  I *think* you're ok.  But @jaredpar is a good resource here.  I think you've satisfied the binary and source compat requirements with your approach, but i'm never 100% sure :)

Reply:
Since Jared is out, I checked with some other binary gurus and this appears sufficient for both binary and source compat.

---
In reply to: [301330874](https://github.com/dotnet/roslyn/pull/37048#discussion_r301330874) [](ancestors = 301330874)

Reply:
Thanks

---
In reply to: [302303932](https://github.com/dotnet/roslyn/pull/37048#discussion_r302303932) [](ancestors = 302303932,301330874)